src/opm/input/eclipse/Schedule/ScheduleDeck.cpp

```diff
@@ -332,6 +342,13 @@ void ScheduleDeck::add_block(ScheduleTimeType time_type, const time_point& t, Sc
 void ScheduleDeck::add_TSTEP(const DeckKeyword& TSTEPKeyword, ScheduleDeckContext& context) {
     const auto &item = TSTEPKeyword.getRecord(0).getItem(0);
     for (size_t itemIndex = 0; itemIndex < item.data_size(); itemIndex++) {
+        {
+            const auto tstep = item.get<double>(itemIndex);
+            if (tstep < 0) {
+                const auto msg = fmt::format("a negatvie TSTEP value {} is input", tstep);
```

Comment:
Misprint: "negatVIe" versus "negatIVe".

Reply:
Thanks for the good catch! Both comments are addressed.